library/init/coe.lean

```diff
@@ -63,7 +67,7 @@ def coe_t {a : Sort u} {b : Sort v} [has_coe_t a b] : a → b :=
 def coe_fn_b {a : Sort u} [has_coe_to_fun.{u v} a] : Π x : a, has_coe_to_fun.F.{u v} x :=
 has_coe_to_fun.coe
 
-/- User level coercion operators -/
+/-! ## User level coercion operators -/
```

Comment:
[Mathlib style](https://leanprover-community.github.io/contribute/doc.html#sectioning-comments) is to uses 3rd level headings for these:
```suggestion
/-! ### User level coercion operators -/
```